sdk/search/search-documents/README.md

```diff
@@ -84,39 +81,122 @@ const serviceClient = new SearchServiceClient("<endpoint>", new AzureKeyCredenti
+## Examples
 
-**Note**: This client cannot function in the browser because the APIs it calls do not have support for Cross-Origin Resource Sharing (CORS).
+### Create an Index
 
-### Documents
+```js
+const { SearchServiceClient, AzureKeyCredential } = require("@azure/search-documents");
 
-An item stored inside a search index. The shape of this document is described in the index using `Field`s. Each Field has a name, a datatype, and additional metadata such as if it is searchable or filterable.
+const client = new SearchServiceClient("<endpoint>", new AzureKeyCredential("<apiKey>"));
 
-### Pagination
+async function main() {
+  const result = await client.createIndex({
+    name: "example-index",
+    fields: [
+      {
+        type: "Edm.String",
+        name: "id",
+        key: true
+      },
+      {
+        type: "Edm.Double",
+        name: "awesomenessLevel",
+        sortable: true,
+        filterable: true,
+        facetable: true
+      },
+      {
+        type: "Edm.String",
+        name: "description",
+        searchable: true
+      },
+      {
+        type: "Edm.ComplexType",
+        name: "details",
+        fields: [
+          {
+            type: "Collection(Edm.String)",
+            name: "tags",
+            searchable: true
+          }
+        ]
+      },
+      {
+        type: "Edm.Int32",
+        name: "hiddenWeight",
+        hidden: true
+      }
+    ]
+  });
 
-Typically you will only wish to [show a subset of search results](https://docs.microsoft.com/azure/search/search-pagination-page-layout#total-hits-and-page-counts) to a user at one time. To support this, you can use the `top`, `skip` and `includeTotalResultCount` parameters to provide a paged experience on top of search results.
+  console.log(result);
+}
 
-### Document field encoding
+main();
+```
 
-[Supported data types](https://docs.microsoft.com/rest/api/searchservice/Supported-data-types) in an index are mapped to JSON types in API requests/responses. The JS client library keeps these mostly the same, with some exceptions:
+### Upload documents into an index
 
-- `Edm.DateTimeOffset` is converted to a JS `Date`.
-- `Edm.GeographyPoint` is converted to a `GeographyPoint` type exported by the client library.
-- Special values of the `number` type (NaN, Infinity, -Infinity) are serialized as strings in the REST API, but are converted back to `number` by the client library.
+You can upload multiple documents into index inside a batch:
 
-**Note**: Data types are converted based on value, not the field type in the index schema. This means that if you have an ISO8601 Date string (e.g. "2020-03-06T18:48:27.896Z") as the value of a field, it will be converted to a Date regardless of how you stored it in your schema.
```

Comment:
This note was explicitly requested by @brjohnstmsft since it might be unexpected by users

Reply:
I have added another section ```Typescript/Javascript specific concepts``` and added this information back